app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Home controller here.
```

Comment:
Delete unnecessary files. We can create stylesheets for these if we need in future. Though we don't have the convention of using stylesheet per model.